cli/cli.py

```diff
@@ -61,33 +61,33 @@ def find_layout(self, layout, lpf):
                 return os.path.abspath("./layouts/"+layout)
             print("Not found")
             print("Finding layout from upstream")
-            layouts = self.lp_layout_list(lpf["layout_upstream"])
+            layouts = self.lp_layout_list(pf["layout_upstream"])
             layouts = [x["name"] for x in layouts]
             if layout in layouts:
                 print("Found!!")
                 print("Fetching layout from upstream !")
-                self.lp_topo_get(layout, lpf["layout_upstream"])
+                self.lp_topo_get(layout, pf["layout_upstream"])
                 return os.path.abspath("./layouts/"+topology)
             print("Not Found !!")
         except Exception as e:
             print(e)
 
-    def lp_rise(self, lpf, target):
-        lpf = parse_yaml(lpf)
+    def lp_rise(self, pf, target):
+        pf = parse_yaml(pf)
         init_dir = os.getcwd()
         e_vars = {}
         e_vars['linchpin_config'] = self.get_config_path()
         e_vars['outputfolder_path'] = init_dir+"/outputs"
-        e_vars['inventory_outputs_path'] = init_dir+"/inventory"
+        e_vars['inventory_outputs_path'] = init_dir+"/inventories"
```

Comment:
This needs to be a variable stored somewhere. I will address this in the linchpin_config. @samvarankashyap, does that sound reasonable to you?